website/config/routes.js

```diff
@@ -996,6 +996,7 @@ module.exports.routes = {
   //  ╩ ╩╝╚╝═╩╝╩╚═╚═╝╩═╩╝  ╩  ╩╚═╚═╝╩ ╚═ ╩   ╚═╝╝╚╝═╩╝╩  ╚═╝╩╝╚╝ ╩ ╚═╝
   'POST /api/android/v1/signupUrls': { action: 'android-proxy/create-android-signup-url', csrf: false},
   'POST /api/android/v1/enterprises': { action: 'android-proxy/create-android-enterprise', csrf: false},
+  'GET /api/android/v1/enterprises': { action: 'android-proxy/get-android-enterprises', csrf: false},
```

Comment:
CSRF protection does not apply to GET requests.
```suggestion
  'GET /api/android/v1/enterprises': { action: 'android-proxy/get-android-enterprises'},
```